### PR TITLE
Yi authendefication

### PR DESCRIPTION
**Pull request by IlinykhYE**

### CODE REVIEW

File: LTRegistratorApi/LTRegistratorApi/Controllers/AccountController.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                _configuration["JwtIssuer"],
+                claims,
+                expires: expires,
+                signingCredentials: creds
+            );
+
+            return new JwtSecurityTokenHandler().WriteToken(token);
+        }
+
+        public class LoginDto
```
Comment (MrAlexMoskvin) [inline LTRegistratorApi/LTRegistratorApi/Controllers/AccountController.cs]: Please move these two classes into separate class files and put them under existing Models folder.

Provide summary to class and properties.

File: LTRegistratorApi/LTRegistratorApi/LTRegistratorApi.csproj [hunk 2]
```diff
@@ -10,8 +10,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.Identity.Core" Version="2.2.2" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.1.11">
```
Comment (MrAlexMoskvin) [inline LTRegistratorApi/LTRegistratorApi/LTRegistratorApi.csproj]: I am not sure that it is really needed. According to "ASP .NET core in Action" everything is already included in Microsoft.AspNetCore.App. Please try to remove them and start project.

File: LTRegistratorApi/LTRegistratorApi/Migrations/ApplicationContextModelSnapshot.cs [hunk 3]
```diff
@@ -14,13 +14,13 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
```
Comment (MrAlexMoskvin) [inline LTRegistratorApi/LTRegistratorApi/Migrations/ApplicationContextModelSnapshot.cs]: please, delete this file. We don't need migrations for now

File: LTRegistratorApi/LTRegistratorApi/Migrations/20190705171421_productsdb.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (MrAlexMoskvin) [inline LTRegistratorApi/LTRegistratorApi/Migrations/20190705171421_productsdb.cs]: Please delete all files from "Migration" folder. We don't need them for now

File: LTRegistratorApi/LTRegistratorApi/Controllers/AccountController.cs [hunk 5]
```diff
@@ -0,0 +1,113 @@
+﻿using System;
```
Comment (MrAlexMoskvin) [inline LTRegistratorApi/LTRegistratorApi/Controllers/AccountController.cs]: Can I ask you to introduce:
1) Summary to class, fields, properties and methods.
2) Regions for: public methods, private methods, public fields, private fields...

#region private methods
//some code with private methods
#endregion

**Comment by IlinykhYE** [comment]
authendefication and token

**Pull request merged**